commands.js

```diff
@@ -1715,6 +1721,7 @@ var commands = exports.commands = {
 		}
 		this.logEntry(user.name + " used /crashfixed");
 	},
+	crashfixedhelp: ["/crashfixed - Announces that the server has been fixed without the need of a restart. Requires: ~"],
```

Comment:
It's also inaccurate - the main use of `/crashfixed` is to end a server lockdown caused by a crash.